commercetools-models/src/main/java/io/sphere/sdk/producttypes/commands/updateactions/SetInputTip.java

```diff
@@ -0,0 +1,43 @@
+package io.sphere.sdk.producttypes.commands.updateactions;
+
+import io.sphere.sdk.commands.UpdateActionImpl;
+import io.sphere.sdk.models.LocalizedString;
+import io.sphere.sdk.products.attributes.AttributeDefinition;
+import io.sphere.sdk.producttypes.ProductType;
+
+import javax.annotation.Nullable;
+
+/**
+ * Sets the attribute input tip. Allows to set additional information about the specified attribute.
+ *
+ * {@doc.gen intro}
+ *
+ * {@include.example io.sphere.sdk.producttypes.commands.ProductTypeUpdateCommandIntegrationTest#setInputTip()}
+ *
+ * @see AttributeDefinition#getInputTip()
```

Comment:
Adding a link helps to understand the relation between the resource and this update action 👍